klimenko/Lesson14/src/main/java/com/klimenko/Lesson14/dao/ApartmentsDaoSD.java

```diff
@@ -0,0 +1,16 @@
+package com.klimenko.Lesson14.dao;
+
+import com.klimenko.Lesson14.entity.Apartment;
+import com.klimenko.Lesson14.entity.User;
+import org.springframework.data.repository.CrudRepository;
+
+import java.util.List;
+
+public interface ApartmentsDaoSD extends CrudRepository<Apartment, Integer> {
+
+    void deleteByEstateId(int id);
+
+    Apartment findApartmentByEstateId(int id);
+
+    List<Apartment> findAll();
```

Comment:
findAll is already in CureRepository, but it's return Iterable, so you need only to cast Iterable to List